### DISCUSSION TITLE
Русская озвучка  // G2 NotR

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/gothic2_steam_fix#27] [General]
https://steamcommunity.com/sharedfiles/filedetails/?id=2251488739

***

распаковал архивы с русской озвучкой в одну папку. получилось вот так

![image](https://user-images.githubusercontent.com/49514764/95342985-acc0ed80-08c0-11eb-9e67-6a986d5c1ae6.png)

у тебя есть вся озвучка в одной папке?  смотрю в твоём инсталлере есть:
- Speech1
- Speech_Addon
- g2a_nr_scriptpatch_v25_speech_add

а у меня есть ещё какой то с русскими словами _Speech_Add_.vdf 

***

**Comment by dosinabox** [comment]
> _Speech_Add_.vdf

это бывший g2a_nr_scriptpatch_v25_speech_add.mod, со времен до кастомного мод-стартера. Можешь удалить.

**Comment by Kor-Angar** [comment]
https://yadi.sk/d/aP_nr4Cv6nzfrQ?w=1

я тогда на всякий случай ссылку положу на архив :-)
https://yadi.sk/d/PrEekKkOdkmCbw

**Comment by dosinabox** [comment]
Speech1 + Speech_Addon = 20849 файлов

**Comment by Kor-Angar** [comment]
> Speech1 + Speech_Addon = 20849 файлов

а с g2a_nr_scriptpatch_v25_speech_add ?

**Comment by dosinabox** [comment]
+70

**Comment by Kor-Angar** [comment]
понятно, что не понятно

**Comment by dosinabox** [comment]
а что не понятно? цифры не сходятся?

**Comment by Kor-Angar** [comment]
вот если взять к примеру всю русскую озвучку с учётом патча, сколько получится файлов?

**Comment by dosinabox** [comment]
В самой крайней сборке от 3 сентября - 20919.

**Comment by Kor-Angar** [comment]
> В самой крайней сборке от 3 сентября - 20919.

это фактическая цифра или расчётная?

**Comment by Kor-Angar** [comment]
меня вот цифра итоговая интересует. хотелось бы сделать один архив с озвучкой русскай в виде мода, что бы его можно было включать и выключать когда хочется

**Comment by dosinabox** [comment]
Фактическая.
15497+5352+70

**Comment by Kor-Angar** [comment]
> Фактическая.
> 15497+5352+70

именно так на выборах считают проценты согласно закону :-)

 фактическая это когда все файлы окажутся в одной папке.

**Comment by Kor-Angar** [comment]
давай подойдём к вопросу с другого боку. Порядок распаковки архивов такой?

1) Speech1.vdf
2) Speech_Addon.vdf
3) Speech_Add.vdf
4) g2a_nr_scriptpatch_v25_speech_add.mod

**Comment by dosinabox** [comment]
> Порядок распаковки архивов такой?

Такой, но

> 3\. Speech_Add.vdf

не нужен вообще, удали

> 4\. g2a_nr_scriptpatch_v25_speech_add.mod

меняется с каждой новой версией обновления, нет смысла его распаковывать

**Comment by Kor-Angar** [comment]
Спасибо за исчерпывающий ответ. Благодаря твоим ответам мне становится понятна логика работы не только скриптов, но и механика реализации демократических основ на пост советском пространстве. 12:35, 8 октября 2020 г., dosinabox <notifications@github.com>:

Порядок распаковки архивов такой?

Такой, но

3. Speech_Add.vdf

не нужен вообще, удали

4. g2a_nr_scriptpatch_v25_speech_add.mod

меняется с каждой новой версией обновления, нет смысла его распаковывать

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by Kor-Angar** [comment]
есть замечания?

https://steamcommunity.com/sharedfiles/filedetails/?id=2251488739

**Comment by dosinabox** [comment]
Сложно, запутанно и непонятно зачем. Но наверное кому-то нужно. Но не обывателю-первопроходцу.
Поменяй местами "как запустить" и "зачем". "Зачем" именно касательно твоего способа здесь ключевое.

**Comment by Kor-Angar** [comment]
```
[INFO]
Title=G2 NotR  Official v.2.6 (RU)
Version=2.6
Authors=RIP Akella
Webpage=https://steamcommunity.com/sharedfiles/filedetails/?id=2251488739
;Description=!<symlink>GothicGame.rtf
Description=!<symlink>G2_NotR_speech_ru.rtf
Icon=Gothic2.exe

[FILES]
VDF=G2_NotR_speech_ru.mod
Game=Content\Gothic
FightAI=Content\Fight
Menu=System\Menu
Camera=System\Camera
Music=System\Music
SoundEffects=System\SFX
ParticleEffects=System\ParticleFX
VisualEffects=System\VisualFX
OutputUnits=OU

[SETTINGS]
Player=PC_HERO
World=NewWorld\NewWorld.zen

[OPTIONS]
show_Info=0
show_InfoX=800
show_InfoY=7200
show_Version=1
show_VersionX=6500
show_VersionY=7200
show_Focus=1
show_FocusItm=1
show_FocusMob=1
show_FocusNpc=1
show_FocusBar=1
force_Subtitles=0
force_Parameters=

[OVERRIDES]
INTERNAL.extendedMenu=1
```

> Сложно, запутанно и непонятно зачем. Но наверное кому-то нужно. Но не обывателю-первопроходцу.

100 человек зайдёт, 50 уйдет на твой топик, 3 скачает, 1 оценит. Это тоже самое что ты делаешь только наоборот. 

> Поменяй местами "как запустить" и "зачем". "Зачем" именно касательно твоего способа здесь ключевое

это только кажется. там же дело в распределение кликов. поэтому с точки зрения кликов всё верно. а ключевые мысли на Стимах не читают ;-)

**Comment by dosinabox** [comment]
В целом, я против такого дробления аудитории. Было одно руководство, добавилось ещё три, которые делают то же самое, но наоборот, или на другом языке, или для какой-то определенной аудитории. И тут уже как повезет с индексацией и выдачей в поиске. Кому-то попадется единое решение, кому-то только стартер, кому-то только русская озвучка (без русского текста). Возвращаемся на 5 лет назад, когда собирали себе игру по кусочкам с разных ресурсов.
Но запретить не могу, так что пусть будет.

**Comment by Kor-Angar** [comment]
ту можешь быть дробления аудитория. но она происходит до сих пор. и будет происходить. люди же субъективны, люди разные. по разному на всё это смотрят, по разному читают. я например твои топики через яндекс не нахожу. зато нахожу какие то другие материалы в которых ты из года в год пытаешься что то объяснить. опять же твоя сборка она такая слишком правильная, ты для себя правду в такой вот конфигурации определила и пытаешься её насадить. может твоя правда и хорошо, но как это хорошо понять то? :-) Топика с неофициальным патчем на стими у тебя нет, хотя это важный и полноценный продукт, который фактически никак несвязан со СтимФиксом. СтимФикса без неофициального патча это или архив на 600 мб или надо догадаться что он в разделе интернационал. версии без озвучки на стими тоже нет, надо всё равно  качать 600 мб.  Я рад что ты сделала Стартер и теперь хотя бы в рамках твоего русского стим фикса можно запустить оригинальные скрипты.

> Возвращаемся на 5 лет назад

не всё возвращается. скорее есть люди которые там и находятся ;-) 

> Но запретить не могу, так что пусть будет.

а если бы могла? :-)

**Comment by dosinabox** [comment]
Народу не нравится ЯД :)

![Screenshot_1](https://user-images.githubusercontent.com/39790055/95957073-e5545000-0e07-11eb-824e-1876d8f213e5.png)

**Comment by Kor-Angar** [comment]
поэтому и надо делать зеркало. кому не нравится у кого то не работает или просто что то упало. ЯД может не нравится, но он даёт хоть какую то статистику. Про ограничения не слышал.

**Comment by dosinabox** [comment]
> Про ограничения не слышал.

После определенного количества общих загрузок файл можно скачать только сначала сохранив в свой ЯД, что требует регистрации.

**Comment by Kor-Angar** [comment]
а можешь ссылку на такой архив бросить? пытаюсь сам такое найти не получается

![image](https://user-images.githubusercontent.com/49514764/95966999-6fa2b100-0e14-11eb-913c-55d4e724717f.png)

**Comment by dosinabox** [comment]
https://yadi.sk/d/bLA7CXF7CY3vg

![image](https://user-images.githubusercontent.com/39790055/95967281-c01a0e80-0e14-11eb-9a8a-d80a667faf99.png)

**Comment by Kor-Angar** [comment]
до такого момента ещё надо дожить :-)

**Comment by Kor-Angar** [comment]
что бы не потерялась https://steamcommunity.com/sharedfiles/filedetails/?id=2251488739
